runtime/runtime/src/ext.rs

```diff
@@ -140,13 +154,18 @@ impl<'a> RuntimeExt<'a> {
             .collect()
     }
 
-    fn append_action(&mut self, receipt_index: u64, action: Action) {
-        self.action_receipts
+    fn append_action(&mut self, receipt_index: u64, action: Action) -> usize {
+        let actions = &mut self
+            .action_receipts
             .get_mut(receipt_index as usize)
             .expect("receipt index should be present")
             .1
-            .actions
-            .push(action);
+            .actions;
+
+        actions.push(action);
+
+        // Return index that action was inserted at
```

Comment:
This should be a function doc-comment.

Reply:
Added comment to the function, but kept this. Would you prefer if I removed it?

Reply:
I think its fine either way, thanks!